composer.json

```diff
@@ -7,11 +7,12 @@
         }
     },
     "require-dev": {
-        "phpunit/phpunit": "3.7.*"
+        "phpunit/phpunit": "^7.0"
     },
     "require": {
-        "symfony/property-access": "2.*|3.*",
-        "symfony/validator": "2.*|3.*",
+        "php": "^7.1",
```

Comment:
Please leave 7.0 support. I don't see where you use 7.1 features except `setUp` in tests. To have maximum backwards compatibility you should try to use lowest appropriate version.